sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -394,6 +395,10 @@ def get_pvc_spec(
         ),
     )
 
+    # If PyTorchJob has 1 worker, ReadWriteOnce access mode is sufficient for PVC.
+    if num_workers == 1:
+        pvc_spec.spec.access_modes = ["ReadWriteOnce"]
```

Comment:
can we take this in storage config? And use it in line 391 instead of this

Reply:
Good point, let me change it.